source/standards/tracking-dependencies.html.md.erb

```diff
@@ -20,13 +20,18 @@ Update your dependencies frequently rather than in ‘big bang’ batches. This
 
 There are tools which scan GitHub repositories and raise PRs when they find dependency updates. Teams at GDS are using:
 
-* [Dependabot][] - used by GOV.UK, GOV.UK Pay and GovWifi. The GOV.UK team manual contains [guidance on using Dependabot][] and [how the PRs raised should be reviewed][]
+* [Dependabot][] - used by GOV.UK, GOV.UK Pay, GovWifi and Digital Marketplace. The GOV.UK docs contain [guidance on using Dependabot][] and [how the PRs raised should be reviewed][]
+    > Note: this is separate from the [security-only updates provided automatically by GitHub Dependabot].
+
+    > Note: repos requiring at least one approving review for PRs cannot, and should not, use [Dependabot's auto-approve-and-merge facility].
+
+    > Note: our account is setup to "Treat PR approval as a request to merge", which helps to reduce some of the day-to-day, manual overhead.
```

Comment:
I've been thinking about this, and I think I'm against it because:

* It provides very limited productivity benefits
* It adds surprising behaviour to GitHub's user interface
* I can imagine a scenario where this confusing behaviour could cause someone who didn't understand the setup to deploy something they didn't mean to

## It provides very limited productivity benefits

As far as I can tell, all this setting does for us is save you from having to click merge after you click approve. We could just train developers to click a couple more buttons, and get all the same benefits without any of the weirdness.

## It adds surprising behaviour to GitHub's user interface

"Approving" a PR in most cases signals that this user is happy for the PR to be merged (perhaps by another user), but having this automated is still surprising. New developers who've worked with Dependabot before will probably not expect this behaviour. People who are used to approving non-dependabot PRs (which won't auto-merge) will have to learn that dependabot PRs behave slightly differently.

If we learn anything from aeroplane crash investigations, it's that surprising user interfaces can cause unexpected bad things to happen.

## I can imagine a scenario where this confusing behaviour could cause someone who didn't understand the setup to deploy something they didn't mean to

Scenario:

> This version bump is fine to go out in principle, but it shouldn't be released until after this [other condition is met], or [something terrible] will happen. I'll approve the PR with a comment saying that it should only be merged once [other condition is met].

Consequence:

> [something terrible] will happen

It's easy to say that this hypothetical user is not using the "approvals" feature properly, but in a blameless incident retro for this hypothetical incident I would point at the approvals user interface as a cause.